packages/react-composites/README.md

```diff
@@ -16,12 +16,12 @@ rush update
 
 #### Create `.env` file
 
-The webpack bundler for test react app reads environment variables from the `.env` file and assigns them to the `process.env` variable.
-These variables can then be accessed inside the code using `process.env.{VAR_NAME}`
+Create a `.env` file inside `tests/browser` directory.
 
-Create a `.env` file inside `tests/browser/app` directory.
+The E2E browser tests require a valid connection string to run. This connection string is provided to the test runner through environment variables.
+The environment variables inside a `.env` file are loaded at the beginning of E2E tests.
 
-Sample `tests/browser/app/.env` file
+Sample `tests/browser/.env` file
```

Comment:
Should we add a `.env` file that has CONNECTION_STRING=REPLACE_WITH_CONNECTION_STRING ?

Reply:
So its a similar experience to running the samples

Reply:
.env files should never be committed to the source.
That would defeat the purpose of using a .env file.